homeassistant/components/screenlogic/binary_sensor.py

```diff
@@ -49,26 +49,34 @@ class ScreenLogicPushBinarySensorDescription(
         data_root=(DEVICE.CONTROLLER, GROUP.SENSOR),
         key=VALUE.ACTIVE_ALERT,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="active_alert",
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.STATUS_CHANGED,
         data_root=(DEVICE.CONTROLLER, GROUP.SENSOR),
         key=VALUE.CLEANER_DELAY,
+        translation_key="circuit_delay",
+        translation_placeholders={"circuit": "Cleaner"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.STATUS_CHANGED,
         data_root=(DEVICE.CONTROLLER, GROUP.SENSOR),
         key=VALUE.FREEZE_MODE,
+        translation_key="freeze_mode",
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.STATUS_CHANGED,
         data_root=(DEVICE.CONTROLLER, GROUP.SENSOR),
         key=VALUE.POOL_DELAY,
+        translation_key="circuit_delay",
+        translation_placeholders={"circuit": "Pool"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.STATUS_CHANGED,
         data_root=(DEVICE.CONTROLLER, GROUP.SENSOR),
         key=VALUE.SPA_DELAY,
+        translation_key="circuit_delay",
+        translation_placeholders={"circuit": "Spa"},
```

Comment:
Why don't we also translate the names of the circuits? Cleaner, Pool and Spa sounds quite translatable